mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -209,20 +251,27 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_7967 isUsedOnRelease ()Z
 	METHOD method_7968 isEqual (Lnet/minecraft/class_1799;)Z
 		ARG 1 stack
-	METHOD method_7969 getTag ()Lnet/minecraft/class_2487;
+	METHOD method_7969 getNbt ()Lnet/minecraft/class_2487;
+		COMMENT {@return the custom NBT of this item stack, may be {@code null}}
+		COMMENT
+		COMMENT @see #hasNbt()
+		COMMENT @see #getOrCreateNbt()
+		COMMENT @see #setNbt(NbtCompound)
 	METHOD method_7970 damage (ILjava/util/Random;Lnet/minecraft/class_3222;)Z
 		ARG 1 amount
 		ARG 2 random
 		ARG 3 player
 	METHOD method_7971 split (I)Lnet/minecraft/class_1799;
 		ARG 1 amount
 	METHOD method_7972 copy ()Lnet/minecraft/class_1799;
+		COMMENT Creates and returns a copy of this item stack.
 	METHOD method_7973 areEqual (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
 		ARG 0 left
 		ARG 1 right
 	METHOD method_7974 setDamage (I)V
 		ARG 1 damage
-	METHOD method_7975 areTagsEqual (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
+	METHOD method_7975 areNbtEquals (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
```

Comment:
This compares 2 objects for equality, not the current object and another - it should be `areNbtEqual` or `areNbtsEqual`